pom.xml

```diff
@@ -13,7 +13,7 @@
   <name>ohsome API</name>
   <description>A public Web-RESTful-API for "ohsome" OpenStreetMap history data.</description>
   <packaging>jar</packaging>
-  <version>1.2.0-SNAPSHOT</version>
+  <version>1.2.1</version>
```

Comment:
ok.. why was the master still on 1.2.0-SNAPSHOT? :D

I assume it should change to 1.3.0-SNAPSHOT, right?

Reply:
```suggestion
  <version>1.3.0-SNAPSHOT</version>
```